OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -159,5 +194,41 @@ void LoadMod(ModMetadata mod)
 				Game.InitializeMod(mod.Id, null);
 			});
 		}
+
+		static bool IsModInstalled(ModMetadata mod)
+		{
+			var manifest = new Manifest(mod.Id);
+			var folders = manifest.Folders.Where(f => f.Contains("^")).Select(f => f.Substring(f.IndexOf("^", StringComparison.Ordinal)));
+
+			var filesList = new List<string>();
+			foreach (var folder in folders)
+			{
+				var path = Platform.ResolvePath(folder).Replace("/", "\\");
+				if (!Directory.Exists(path))
+					continue;
+
+				var filesInDir = Directory.GetFiles(path).Select(f => f.ToLower());
```

Comment:
This isn't just the files, but the file + folder. It is `~/.openra/Content/` on Unix so making this `~/.openra/content/` will break on case sensitive file systems. Change it to:

```csharp
var filesInDir = Directory.GetFiles(path);
```